packages/starlight/components/Icons.ts

```diff
@@ -126,6 +126,14 @@ export const BuiltInIcons = {
+	deno: 
+		'<path d="M128 0c70.692 0 128 57.308 128 128c0 70.692-57.308 128-128 128C57.308 256 0 198.692 0 128C0 57.308 57.308 0 128 0"/><path fill="#fff" d="M123 72.45c17.9 0 33.25 4.95 45.35 14.4c10.184 7.96 17.68 18.9 21.617 31.278l.133.422l.1.3l.1.35l.2.65l.3 1.2l.8 2.8l.85 3.2l3.75 13.7l3.7 13.8l4.2 15.7l6.75 25.25l2.55 9.5l-.55.6c-17.622 19.256-41.713 32.483-68.827 36.288l-.823.112l-.25-1.65l-.45-3.3l-.4-2.4l-.45-3.15l-.6-3.85l-.25-1.5l-.55-3.65l-.35-2.15l-.45-2.8l-.45-2.7l-.45-2.6l-.45-2.55l-.4-2.5l-.45-2.4l-.4-2.3l-.3-1.7l-.35-1.65l-.6-3.15l-.3-1.5l-.35-1.85l-.3-1.3l-.25-1.25l-.25-1.2l-.15-.8l-.35-1.5l-.5-2.2l-.2-.7l-.25-1l-.2-.95l-.25-.95l-.25-.9l-.15-.55l-.25-.85l-.2-.8l-.15-.55l-.2-.5l-.15-.45l-.2-.7l-.15-.5l-.1-.3a33.4 33.4 0 0 0-.706-1.886l-.194-.464l-.15-.3l1.15-3l-4.55.15l-1.25.05c-41.3.85-67.95-16.7-67.95-44.2c0-29.15 29-52.6 66.2-52.6m-40.3 137.7c.935-3.2 4.246-5.091 7.45-4.29l.15.04c3.25.837 5.19 4.098 4.485 7.348l-.035.152l-.05.15l-6.05 22.55l-.85-.35a114.35 114.35 0 0 1-9.94-4.253l-.81-.397l5.6-20.8zm34.8-15.25c.95-3.25 4.35-5.15 7.65-4.25a6.25 6.25 0 0 1 3.815 2.991l.085.159l.2.9l.3 1.45l.2 1.05l-.05.25l-.15.7v.15l-8.5 31.5l-.05.15a6.25 6.25 0 0 1-12.08-3.098l.03-.152v-.15l8.5-31.5zm-51.6-36.7c3.024 2.64 6.509 5.05 10.322 7.096l.478.254l-7.7 28.6l-.05.15a6.25 6.25 0 0 1-12.084-3.149L56.9 191l.05-.15l8.5-31.5l.05-.2zm-27.9-32c.95-3.2 4.35-5.1 7.65-4.25c3.2.886 5.14 4.148 4.435 7.398l-.035.152v.15l-8.5 31.5l-.05.15a6.25 6.25 0 0 1-12.084-3.149l.034-.151v-.15l8.5-31.5zm190.6-7.15c.95-3.2 4.35-5.1 7.6-4.25c3.25.886 5.19 4.148 4.485 7.35l-.035.15v.2L232.1 154v.15a6.271 6.271 0 0 1-12.139-3.15l.039-.15l.05-.15l8.5-31.5zM27.1 72.75l.217.007c.433.02.863.085 1.283.193a6.28 6.28 0 0 1 4.485 7.398l-.035.152l-.05.15l-8.5 31.5l-.05.15c-.95 3.2-4.35 5.1-7.6 4.25a6.2 6.2 0 0 1-3.1-1.9a114.2 114.2 0 0 1 13.033-41.318zm179.95 4.45c1-3.2 4.35-5.1 7.65-4.25c3.2.886 5.188 4.148 4.485 7.398l-.035.152l-.05.15l-8.5 31.5l-.05.15a6.25 6.25 0 0 1-12.084-3.149l.034-.151l.05-.15l8.5-31.5zM56.5 47.25c.95-3.2 4.35-5.1 7.6-4.25c3.25.886 5.19 4.148 4.485 7.35l-.035.15l-.05.2L60 82.2l-.05.15a6.25 6.25 0 0 1-12.084-3.149l.034-.151l.05-.15l8.5-31.5zm109.2 5.95c.95-3.2 4.35-5.1 7.65-4.25c3.2.886 5.14 4.148 4.435 7.398l-.035.152v.15l-6.7 24.75l-.55-.45a69.95 69.95 0 0 0-9.935-6.094l-.615-.306l5.7-21.2zm-47.95-39.75a6.2 6.2 0 0 1-.004 2.603l-.046.197l-.05.15l-8.5 31.5l-.05.15a6.25 6.25 0 0 1-12.08-3.098l.03-.152l.05-.15L105 15.3l.85-.15a114.95 114.95 0 0 1 11.9-1.7m78.2 21.75l.75.55a115.6 115.6 0 0 1 9.222 7.666l.628.584l-.2.65l-.05.15a6.25 6.25 0 0 1-12.08-3.098l.03-.152l.05-.15zM146.5 14.5l.9.15a114.3 114.3 0 0 1 10.44 2.267l.86.233l-3.15 11.75l-.05.15a6.25 6.25 0 0 1-12.08-3.098l.03-.152l.05-.15z"/><path d="M131 93.5a8 8 0 1 1 0 16a8 8 0 0 1 0-16"/>',
```

Comment:
A few changes needed here:

- paths should only specify the `d` attribute (or other attributes related to path rendering) not styling attributes like `class` or `fill` etc.
- compressed a few of these icons down a bit smaller with [SVGOMG](https://jakearchibald.github.io/svgomg/)
- the icons need to be 24 x 24 pixels, but the Deno path here was much larger, meaning it was displaying empty: <img width="138" alt="empty box labelled Deno" src="https://github.com/withastro/starlight/assets/357379/6362be05-e1b8-46c7-9b65-5c2b7a19f2e1">, but I fixed that


```suggestion
	node: '<path d="M12 23.96c-.34 0-.66-.1-.96-.25l-3.03-1.73c-.45-.25-.22-.33-.09-.38.62-.2.73-.24 1.37-.6.07-.04.16-.02.23.03l2.32 1.34c.1.05.2.05.27 0l9.1-5.08a.27.27 0 0 0 .13-.24V6.9c0-.11-.05-.2-.14-.24L12.11 1.6c-.09-.05-.2-.05-.27 0L2.75 6.66c-.09.04-.13.15-.13.24v10.14c0 .1.04.2.13.25l2.49 1.38c1.34.66 2.18-.1 2.18-.88V7.78c0-.13.12-.26.28-.26h1.16c.13 0 .27.1.27.26v10.01c0 1.74-.98 2.75-2.69 2.75-.52 0-.93 0-2.1-.55l-2.38-1.32a1.85 1.85 0 0 1-.96-1.6V6.92c0-.66.36-1.28.96-1.6l9.08-5.1a2.1 2.1 0 0 1 1.92 0l9.08 5.08c.6.33.96.95.96 1.61v10.15c0 .66-.36 1.27-.96 1.6l-9.08 5.09a2.4 2.4 0 0 1-.96.2m2.8-6.98c-3.98 0-4.8-1.76-4.8-3.26 0-.13.11-.26.27-.26h1.18c.14 0 .25.08.25.22.19 1.16.71 1.73 3.12 1.73 1.92 0 2.74-.41 2.74-1.4 0-.58-.23-1-3.21-1.28-2.49-.24-4.04-.77-4.04-2.68 0-1.79 1.55-2.84 4.15-2.84 2.91 0 4.35.96 4.53 3.08a.35.35 0 0 1-.07.2.27.27 0 0 1-.18.08h-1.18a.27.27 0 0 1-.25-.2c-.28-1.2-.98-1.6-2.85-1.6-2.1 0-2.35.7-2.35 1.23 0 .64.3.84 3.12 1.19 2.8.35 4.13.86 4.13 2.75-.03 1.94-1.67 3.04-4.56 3.04"/>',
	cloudflare:
		'<path d="M16.5 16.84c.16-.5.1-.97-.15-1.3-.22-.33-.6-.5-1.06-.53l-8.66-.11a.15.15 0 0 1-.13-.08.21.21 0 0 1-.02-.15.25.25 0 0 1 .2-.15l8.74-.12a3.13 3.13 0 0 0 2.55-1.91l.5-1.3a.25.25 0 0 0 .01-.17 5.68 5.68 0 0 0-10.93-.59 2.58 2.58 0 0 0-3.35.24 2.55 2.55 0 0 0-.67 2.44 3.64 3.64 0 0 0-3.5 4.17.18.18 0 0 0 .17.15h15.98a.22.22 0 0 0 .21-.16l.12-.43Zm2.77-5.56-.24.01c-.06 0-.1.05-.13.1l-.34 1.18c-.15.5-.1.97.16 1.31.22.32.6.5 1.06.52l1.84.12c.06 0 .1.02.14.07.02.04.03.1.02.15a.23.23 0 0 1-.2.15l-1.93.12a3.11 3.11 0 0 0-2.55 1.91l-.14.36c-.03.07.02.14.1.14h6.6a.18.18 0 0 0 .16-.12 4.74 4.74 0 0 0-4.56-6v-.02Z"/>',
	vercel: '<path d="m12 1l12 21H0z"/>',
	netlify:
		'<path d="M6.49 19.04h-.23L5.13 17.9v-.23l1.73-1.71h1.2l.15.15v1.2L6.5 19.04ZM5.13 6.31V6.1l1.13-1.13h.23L8.2 6.68v1.2l-.15.15h-1.2zm9.96 9.09h-1.65l-.14-.13v-3.83c0-.68-.27-1.2-1.1-1.23c-.42 0-.9 0-1.43.02l-.07.08v4.96l-.14.14H8.9l-.13-.14V8.73l.13-.14h3.7a2.6 2.6 0 0 1 2.61 2.6v4.08l-.13.14Zm-8.37-2.44H.14L0 12.82v-1.64l.14-.14h6.58l.14.14v1.64zm17.14 0h-6.58l-.14-.14v-1.64l.14-.14h6.58l.14.14v1.64zM11.05 6.55V1.64l.14-.14h1.65l.14.14v4.9l-.14.14h-1.65zm0 15.81v-4.9l.14-.14h1.65l.14.13v4.91l-.14.14h-1.65z"/>',
	deno: 
		'<path d="M12 0a12 12 0 1 1 0 24 12 12 0 0 1 0-24m-.47 6.8c-3.49 0-6.2 2.19-6.2 4.92 0 2.58 2.5 4.23 6.37 4.15h.12l.42-.02-.1.28v.03l.09.22v.03l.02.04.02.07.02.04.01.05.02.05.02.07.02.08.02.06.02.08.02.09.02.09.03.1.01.06.03.1.02.1.03.15.02.07.02.11.03.12.02.12.04.17.02.15.04.2.02.1.03.15.03.15.04.22.04.23.04.23.04.24.04.24.04.26.04.26.04.2.05.34.02.14.06.36.04.3.04.22.04.31.03.16a10.76 10.76 0 0 0 6.53-3.41l.05-.06-.24-.89-.64-2.37-.39-1.47-.35-1.3-.21-.78-.14-.5-.08-.3-.07-.26-.03-.11-.02-.07-.01-.03v-.03a6.04 6.04 0 0 0-2.05-2.97 6.75 6.75 0 0 0-4.25-1.35M8.47 19.3a.59.59 0 0 0-.72.4v.01l-.53 1.96q.5.24 1.01.43l.08.03.57-2.11V20a.59.59 0 0 0-.41-.7m3.26-1.43a.59.59 0 0 0-.71.4v.01l-.8 2.96v.01a.59.59 0 0 0 1.12.3l.8-2.96v-.02l.02-.06v-.02l-.02-.1-.02-.14-.02-.08a.58.58 0 0 0-.37-.3Zm-5.55-3.04a1 1 0 0 0-.04.09v.02l-.8 2.95v.02a.59.59 0 0 0 1.13.3v-.01l.72-2.68a5.3 5.3 0 0 1-1.01-.7Zm-1.9-3.4a.59.59 0 0 0-.72.4v.02l-.8 2.95v.01a.59.59 0 0 0 1.13.3l.8-2.96v-.01a.59.59 0 0 0-.41-.7m17.87-.68a.59.59 0 0 0-.72.4v.02l-.8 2.95v.01a.59.59 0 0 0 1.13.3l.8-2.96v-.01a.59.59 0 0 0-.41-.7M2.55 6.81a10.7 10.7 0 0 0-1.26 3.93.59.59 0 0 0 1-.22v-.02l.8-2.95v-.01a.59.59 0 0 0-.55-.73m17.59.02a.59.59 0 0 0-.72.4v.01l-.8 2.96v.01a.59.59 0 0 0 1.13.3l.8-2.96v-.01a.59.59 0 0 0-.41-.7Zm-7.85 1.93a.75.75 0 1 1 0 1.5.75.75 0 0 1 0-1.5M6.01 4.03a.59.59 0 0 0-.71.4v.02L4.5 7.4v.01a.59.59 0 0 0 1.12.3l.8-2.96v-.01a.59.59 0 0 0-.41-.7Zm10.24.56a.59.59 0 0 0-.71.4V5L15 7q.52.26.99.6l.05.04.62-2.32V5.3a.59.59 0 0 0-.41-.7m-5.21-3.34a11 11 0 0 0-1.12.16l-.07.01L9.1 4.2v.01a.59.59 0 0 0 1.13.3l.8-2.96v-.01a.6.6 0 0 0 0-.27m7.34 2.04-.16.58v.02a.59.59 0 0 0 1.13.3V4.2l.02-.07a11 11 0 0 0-.92-.77zm-4.64-1.94-.28 1.05a.59.59 0 0 0 1.13.31v-.01l.3-1.1q-.52-.15-1.06-.24z"/>',
```

Reply:
Planning to start getting the next release ready — so hope you don’t mind me making this change myself!